src/proxyClient.js

```diff
@@ -97,7 +97,14 @@ var frameId = 0;
 
 // Worker
 
-var worker = new Worker('{{{ filename }}}.js');
+var filename = '{{{ filename }}}.js';
+var workerURL = filename;
+var fileBytes = tryParseAsDataURI(filename);
+if (fileBytes) {
+  workerURL = URL.createObjectURL(new Blob([fileBytes], {type: 'application/javascript'}));
+}
+
+var worker = new Worker(workerURL);
```

Comment:
Do we desire the "general" solution of having the 8 instances of `tryParseAsDataURI` in the various read functions? I.e. is that desirable? Or would it be preferable to "hoist" the base64->blob handling to the location of creation, after all if one does a non-single file build, there is only the memory initializer that would be base64 (one location of needed use).

Perhaps the base64 memory initialization method could be a `MEM_INIT_METHOD==3` or similar (and `-s SINGLE_FILE=1` would imply `MEM_INIT_METHOD==3`?). And the base64 handling code could be conditional to `#if SINGLE_FILE`? Then `MEM_INIT_METHOD==0` would still utilize the array initializer, which needs less runtime support.